src/components/Navbar.js

```diff
@@ -0,0 +1,49 @@
+import { NavLink } from 'react-router-dom';
+import '../css/Navbar.css';
+import './styles.css';
+
+function Navbar() {
+  return (
+    <nav className="nav-bar">
+      <h1>Math Magicians </h1>
+      <ul className="nav-ul">
+        <li className="nav-child">
+          <NavLink
+            className="nav-link"
+            style={({ isActive }) => ({
+              color: isActive ? '#fed136' : '#fff;',
+            })}
+            to="/"
+          >
+            Home
+          </NavLink>
+        </li>
+
+        <li className="nav-child">
+          <NavLink
+            className="nav-link"
+            style={({ isActive }) => ({
+              color: isActive ? '#fed136' : '#fff;',
+            })}
+            to="/calculator"
+          >
+            Calculator
+          </NavLink>
+        </li>
```

Comment:
- [ ] Kindly check your `Calculator` component and try to see how you can structure the contents as contained in the wireframe. We have 2 contents aligned side by side - hopefully with `CSS Flexbox` to strengthen your CSS skill while ensuring commitment to aesthetic or good designs which will also help to convince recruiters about your work.
 - Please see the second image in the picture below :point_down: 
![image](https://github.com/yusufyusie/math-magicians/assets/99715640/206b7439-352f-4f73-a323-ea8d520002e5)

Your implementation:
![image](https://github.com/yusufyusie/math-magicians/assets/99715640/d936fd1b-1fea-42b0-b72f-6333cf4cc42c)

- [ ] [OPTIONAL] I think it will be really nice if the calculator can be resized to make sure that users do not have to scroll before interacting fully with your calculator. This is for convenience and a better experience :-1: